### ISSUE TITLE
False positive on interface inheriting from antoher interface

### ISSUE THREAD

**Issue by drank40** [duaraghav8/Ethlint#296]
**Description**

When declaring an interface that inherits from another interface the linter throws a false positive (the program compiles in solidity 0.8)

**Steps to reproduce**

The following code is a simplified version of the code in my dApp

```
1 interface B {
2    ...
3 }
4
5 interface A is B {
6    ...
7 }
```
Line 5 throws the error

*Syntax error: unexpected token i []*

**Expected behavior**

No errors, as this is valid solidity code

**Operating System**

Linux (Arch, kernel 5.18.14)

**Linter version**

Solium version 1.2.5

**Priority**

Not a very common pattern to come across, but still very annoying as I use this code in my dApp
